src/current/_includes/releases/v24.1/v24.1.10.md

```diff
@@ -0,0 +1,95 @@
+      1. A string that can be parsed as valid JSON and deserialized into a map of issuer URLs to corresponding JWKS URIs. In the third case, we will be overriding the JWKS URI present in the issuer's well-known endpoint, e.g., `'{ "issuer_jwks_map": { "https://accounts.google.com": "https://www.googleapis.com/oauth2/v3/certs", "example.com/adfs": "https://example.com/adfs/discovery/keys" }}'`. When `issuer_jwks_map` is set, we directly use the JWKS URI to get the key set. In all other cases where `JWKSAutoFetchEnabled` is set, we obtain the JWKS URI first from the issuer's well-known endpoint and then use this endpoint.
+
+<h3 id="v24-1-10-general-changes">General changes</h3>
+
+- To improve the granularity of changefeed pipeline metrics, the changefeed metrics `changefeed.admit_latency` and `changefeed.commit_latency` have histogram buckets from `5ms` to `60m` (previously `500ms` to `5m`). The following changefeed metrics have histogram buckets from `5ms` to `10m` (previously `500ms` to `5m`): 
+    - `changefeed.parallel_io_queue_nanos`
+    - `changefeed.parallel_io_result_queue_nanos`
+    - `changefeed.sink_batch_hist_nanos`
+    - `changefeed.flush_hist_nanos`
+    - `changefeed.kafka_throttling_hist_nanos` [#136602][#136602]
+- Added support for multiple seed brokers in the new Kafka sink. [#136746][#136746]
+- Added a new metric (`distsender.rangefeed.catchup_ranges_waiting_client_side`) that counts how many rangefeeds are waiting on the client-side limiter to start performing catchup scans. [#136836][#136836]
+- Added changefeed support for the `mvcc_timestamp` option with the `avro` format. If both options are specified, the Avro schema includes an `mvcc_timestamp` metadata field and emits the row's MVCC timestamp with the row data. [#136017][#136017] [#137594][#137594]
+- Added a no-op `AWS_USE_PATH_STYLE` param for forward compatibility with v24.3. [#137100][#137100]
+
+<h3 id="v24-1-10-sql-language-changes">SQL language changes</h3>
+
+- Added the `legacy_varchar_typing` session setting, which reverts the changes of [#133037](https://github.com/cockroachdb/cockroach/pull/133037) that caused the change in typing behavior described in [#137837](https://github.com/cockroachdb/cockroach/pull/137837). Specifically, it makes type-checking and overload resolution ignore the newly added "unpreferred" overloads. This setting defaults to `on`. [#137921][#137921]
+
+<h3 id="v24-1-10-operational-changes">Operational changes</h3>
+
+- Removed the `sql.auth.resolve_membership_single_scan.enabled` cluster setting. This was added out of precaution in case it was necessary to revert back to the old behavior for looking up role memberships, but this escape hatch has never been needed in practice since this was added in v23.1. [#136121][#136121]
+- Telemetry delivery is now considered successful even in cases where we experience a network timeout. This will prevent throttling in cases outside an operator's control. [#136478][#136478]
```

Comment:
```suggestion
- Telemetry delivery is now considered successful even in cases where CockroachDB experiences a network timeout. This will prevent throttling in cases outside an operator's control. [#136478][#136478]
```